test/acceptance/graphql_resolvers/setup_test.go

```diff
@@ -60,6 +62,8 @@ func testGraphQL(t *testing.T) {
 	// tests with classes that have objects with same uuids
 	t.Run("import test data (near object search class)", addTestDataNearObjectSearch)
 
+	waitForIndexing()
```

Comment:
can we change it to `assertEventuallyWIth` like I did this [here](https://github.com/weaviate/weaviate/blob/main/test/acceptance/graphql_resolvers/setup_test.go#L1108-L1124)? so just to avoid sleeps which tend to convert to flaky tests

Reply:
yes great point